router-tests/freeport/freeport.go

```
@@ -414,6 +415,20 @@ func isPortInUse(port int) bool {
 	ln.Close()
+
+	return isPortAccepting(port)
+}
+
+// isPortAccepting reports whether any listener accepts a connection on the port over
+// loopback, including listeners bound to the wildcard address.
+func isPortAccepting(port int) bool {
+	for _, host := range []string{"127.0.0.1", "::1"} {
+		conn, err := net.DialTimeout("tcp", net.JoinHostPort(host, strconv.Itoa(port)), 200*time.Millisecond)
+		if err == nil {
+			conn.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `conn.Close` is not checked

(errcheck)